dftatom fails to build (tries to put files into the Python package)

```
[dftatom] [ 89%] Built target well_rel
[dftatom] [ 91%] Built target vloc_nonrel
[dftatom] [ 93%] Built target vloc_nonrel_dft
[dftatom] [ 95%] Built target vloc_nonrel_dft_hyp
[dftatom] [100%] Built target dftatom_wrapper
[dftatom] Install the project...
[dftatom] -- Install configuration: "Release"
[dftatom] -- Installing: /tmp/bld2/dftatom/ve4xw2mhei2n/lib/libdftatom.so
[dftatom] -- Set runtime path of "/tmp/bld2/dftatom/ve4xw2mhei2n/lib/libdftatom.so" to "/tmp/bld2/dftatom/ve4xw2mhei2n/lib"
[dftatom] CMake Error at dftatom/cmake_install.cmake:44 (FILE):
[dftatom]   file cannot create directory:
[dftatom]   /tmp/bld2/python/k4nkxa6wnacj/lib/python2.7/site-packages/dftatom.  Maybe
[dftatom]   need administrative privileges.
[dftatom] Call Stack (most recent call first):
[dftatom]   cmake_install.cmake:39 (INCLUDE)
[dftatom] 
[dftatom] 
[dftatom] make: *** [install] Error 1
[dftatom ERROR] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 2
[dftatom ERROR] command failed (code=2); raising
```

So this has been broken from the very beginning. Only we finally enabled write protection again, which made this bug surface.

--- a/pkgs/dftatom.yaml
+++ b/pkgs/dftatom.yaml
@@ -10,7 +10,7 @@ sources:
 build_stages:
 - name: configure
   extra: ['-D WITH_PYTHON:BOOL=ON',
-          '-D PYTHON_INSTALL_PATH:PATH=$ARTIFACT/lib/python2.7/site-packages']
+          '-D PYTHON_INSTALL_PATH:PATH=$ARTIFACT/{{python_site_packages_rel}}']
   build_in_source: true
 
 - name: setup_builddir

